mcp/include/pointing.h

```diff
@@ -28,16 +28,21 @@
 
 #include "channels_tng.h"
 
+// size of the gyro ring buffer, age is in centiseconds
 #define GY_HISTORY_AGE_CS 4400
 
+// nominal "at float" altitude = 100 kft
```

Comment:
100 kft?! Is it even legal to use metric prefixes and imperial units at the same time? :zany_face:

Reply:
I try my best to make sure everyone's head explodes a little